neurons/validator/utils.py

```diff
@@ -57,26 +57,34 @@ def ttl_get_block(self) -> int:
 async def check_uid(dendrite, self, uid, response_times):
     try:
         t1 = time.perf_counter()
-        response = await dendrite(self.metagraph.axons[uid], IsAlive(), deserialize=False, timeout=self.async_timeout)
+        response = await dendrite(
+            self.metagraph.axons[uid],
+            IsAlive(),
+            deserialize=False,
+            timeout=self.async_timeout,
+        )
         if response.is_success:
             response_times.append(time.perf_counter() - t1)
+            self.isalive_dict[uid] = 0
             return True
         else:
-
             try:
+                self.isalive_dict[uid] += 1
```

Comment:
Rename variable to indicate that this is tracking failed isalive attempts. Maybe` isalive_failed_dict`

Reply:
good suggestion, changed to `isalive_failed_count`